src/resetAllEntries.ts

```diff
@@ -0,0 +1,7 @@
+import { IAccessTokenProvider } from 'virgil-sdk';
+import KeyknoxClient from './clients/KeyknoxClient';
+
+export async function resetAllEntries(accessTokenProvider: IAccessTokenProvider) {
```

Comment:
Let's call this `unsafeResetAllEntries` to emphasize the fact that it clears the entries without the private key. Also, please mention this method in the README or show usage example in the `example` folder